### DISCUSSION TITLE
Post creation date

### DISCUSSION THREAD

**Discussion by eduherminio** [mmistakes/minimal-mistakes#2698] [Q&A] [answered]
## Summary

Support both creation and last update dates
![image](https://user-images.githubusercontent.com/11148519/93648442-13ae5d80-fa0b-11ea-92b9-a86472da1898.png)

## Motivation

Having already the option of using `date` and `last_modified_at` in posts, I thought that (at least) when both are used at the same time, the former one could be used to show post creation date (as shown in the picture above), providing the authors with the option of displaying both.

## Drawbacks

If keeping backwards compatibility is desired when only `date` is provided, `date` field would acquire two different meanings when it's alone (updated date) and when it lives together with `last_modified_at` (created date).
Otherwise, it could be a relevant (and undesired?) breaking change.

## Possible implementation (`includes/page__date.html`)

```html
{% if page.date and page.last_modified_at%}
  <p class="page__date"><strong><i class="fas fa-fw fa-calendar-alt" aria-hidden="true"></i> {{ site.data.ui-text[site.locale].created_date_label | default: "Created:" }}</strong> <time datetime="{{ page.date | date_to_xmlschema }}">{{ page.date | date: "%B %-d, %Y" }}</time></p>
{% elsif page.date %}
  <p class="page__date"><strong><i class="fas fa-fw fa-calendar-alt" aria-hidden="true"></i> {{ site.data.ui-text[site.locale].date_label | default: "Updated:" }}</strong> <time datetime="{{ page.date | date_to_xmlschema }}">{{ page.date | date: "%B %-d, %Y" }}</time></p>
{% endif %}
{% if page.last_modified_at %}
  <p class="page__date"><strong><i class="fas fa-fw fa-calendar-alt" aria-hidden="true"></i> {{ site.data.ui-text[site.locale].date_label | default: "Updated:" }}</strong> <time datetime="{{ page.last_modified_at | date: "%Y-%m-%d" }}">{{ page.last_modified_at | date: "%B %-d, %Y" }}</time></p>
{% endif %}
```

This would also require adding a `created_date_label` and associated translations.

**Comment by mmistakes** [comment, marked as answer]
Seems a bit clunky to me. The theme already supports post dates, which is the same thing as a “created” or published date.

https://mmistakes.github.io/minimal-mistakes/docs/configuration/#post-dates

**Comment by eduherminio** [comment]
> Seems a bit clunky to me. The theme already supports post dates, which is the same thing as a “created” or published date.
> 
> https://mmistakes.github.io/minimal-mistakes/docs/configuration/#post-dates

Oh, wasn't aware of that, thanks!
In that case you can pretend this feature request never took place!
